src/elements/ComicScroll.tsx

```diff
@@ -0,0 +1,27 @@
+import ComicCard from "../atoms/ComicCard";
+import { IComicData } from "../models/comicData";
+function ComicScroll({
+  heading,
+  comics,
+}: {
+  heading: string;
+  comics: IComicData[];
+}) {
+  return (
+    <>
+      <h4 className="text-medium font-medium">{heading}</h4>
+      <div className="major-cities-scroll bg-[white] overflow-auto whitespace-nowrap mb-6">
```

Comment:
You don't have to use an arbitrary colour/value for the white background, you can just use `bg-white`, it is predefined/preset in Tailwind already.